### PR TITLE
More lines for various string pools

### PR DESCRIPTION
**Pull request by badlypayedgoon**
## About The Pull Request
Added a few more quotes (mostly to test my setup for future stuff)

## Why It's Good For The Game
More lines do not hurt. Also lowers the chance of it being repetitive.

## Changelog
:cl:
add: Added more things
spellcheck: fixed a few typos
/:cl:

### CODE REVIEW

**Comment by DatBoiTim** [comment]
Wait are these the drunk science quotes? LMAOO

**Comment by Effemu** [comment]
good pr 👍

File: strings/pirates.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
         "Space",
         "Solar",
         "Laser",
-        "Ion"
+        "Ion",
+		"Tachyon"
```
Comment (francinum) [inline strings/pirates.json:13]: Fix indentation in this file in general.

File: strings/traumas.json [hunk 2]
```diff
@@ -137,7 +137,8 @@
         "",
         "IS TIS A BUG??",
         "SI IST A BUGG/",
-        "BUG!!!"
+        "BUG!!!",
+		"INSECT-THINGIE!!"
```
Comment (francinum) [inline strings/traumas.json:132]: Same here

File: strings/valentines.json [hunk 3]
```diff
@@ -99,6 +99,7 @@
 		"A heart-shaped candy that reads: WAG MY TAIL",
 		"A heart-shaped candy that reads: PDA",
 		"A heart-shaped candy that reads: DEVILS KISS",
-		"A heart-shaped candy that reads: CHEMISTRY"
+		"A heart-shaped candy that reads: CHEMISTRY",
+		"A heart-shaped candy that reads: LUV"
```
Comment (francinum) [inline strings/valentines.json:102]: God this copy paste sucks.

But it's also not your fault or problem, just fix the indentation in the file while you're here.

**Review by francinum — CHANGES_REQUESTED** [review]

**Comment by francinum** [comment]
Please write a better changelog.

File: strings/valentines.json (hunk 3 above)
Comment (badlypayedgoon) [reply strings/valentines.json:102]: Is it better now?  Sry, I'm new to github and byond coding.

**Review by francinum — APPROVED** [review]

File: strings/valentines.json (hunk 3 above)
Comment (utilbird) [reply strings/valentines.json:102]: dw, they were commenting about how "A heart-shaped candy that reads:" is repeated every line. You didn't do anything wrong

**Pull request merged**